### ISSUE TITLE
Download specific bands 

### ISSUE THREAD

**Issue by Himesh91** [16EAGLE/getSpatialData#45]
Is there is any provisional to download only specific bands from sentinel 2 such as B05 , B08

Actually i have work in these given bands for getting those i have to download all the data which is a waste of my bandwidth. 

Api in Python has the eligibility to download only the required bands.

**Comment by 16EAGLE** [comment]
Good idea, would be useful. Currently, this is not possible through `getSpatialData` but I am looking into it.

**Comment by Himesh91** [comment]
Is there any update against the issue
